Cargo.toml

```
@@ -19,7 +19,7 @@ include = [
 path = "bindings/rust/lib.rs"
 
 [dependencies]
-tree-sitter = "0.19.4"
+tree-sitter = "0.20.6"
```

could you remove the patch number? I think that's causing problems. Cargo will choose the latest patch anyways.
```suggestion
tree-sitter = "0.20"
```

---

NEVERMIND! I was just an idiot and didn't use the correct branch!! gahhh. This is still work changing though. You don't normally specify the branch.